README.md

```diff
@@ -157,6 +158,22 @@ yarn start
 
 The app and API are both served on  [http://localhost:3345/](http://localhost:3345/)
 
+## Deployment to Vonage Cloud Runtime
+
+You can easily deploy your local branch to Vonage Cloud Runtime (VCR) using the tools in this repository.
+
+First, set up your VCR application at https://developer.vonage.com/en/vonage-cloud-runtime/overview. One you have your application, get your application ID  and set it in your config  `VCR_APP_ID` in the top level `./env` file.
```

Comment:
i think this is just a normal vonage application? if so it can be done via the CLI or the API dashboard. I recommend moving the CLI install first then just having a copy and paste for creating a new application

Reply:
Understood, I have made this more generic and changed the link to point to the dashboard 🙏